<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIAL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrs/resources.qrc">
    <normaloff>:/icons/bial-logo.png</normaloff>:/icons/bial-logo.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="GraphicsView" name="logoView">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="ImageViewer" name="imageViewer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_6"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_images">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>R&amp;ecent images:</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuRecent_labels">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Re&amp;cent labels:</string>
     </property>
    </widget>
    <addaction name="actionOpen_image"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionAddLabel"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_images"/>
    <addaction name="menuRecent_labels"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOverlay">
    <property name="title">
     <string>Overlay</string>
    </property>
    <addaction name="actionToggle_overlay"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionShow_controls_dock"/>
    <addaction name="actionShow_labels_dock"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuLayout">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Layout</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAxial"/>
    <addaction name="actionCoronal"/>
    <addaction name="actionSagittal"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLayout"/>
   <addaction name="menuOverlay"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_image">
   <property name="text">
    <string>&amp;Open image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAddLabel">
   <property name="text">
    <string>&amp;Load label</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCoronal">
   <property name="text">
    <string>&amp;Coronal</string>
   </property>
  </action>
  <action name="actionSagittal">
   <property name="text">
    <string>&amp;Sagittal</string>
   </property>
  </action>
  <action name="actionAxial">
   <property name="text">
    <string>&amp;Axial</string>
   </property>
  </action>
  <action name="actionToggle_overlay">
   <property name="text">
    <string>&amp;Activate/Deactivate</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionShow_controls_dock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Controls dock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShow_labels_dock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Labels dock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>src/graphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QWidget</extends>
   <header>src/imageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrs/resources.qrc"/>
 </resources>
 <connections/>
</ui>
